server/lib/fbApi.js

```diff
@@ -117,20 +117,29 @@ const introspect = ({id = null} = {}) => {
 	.then(results => results.metadata);
 };
 
-const post = ({published = false, html = ''} = {}) => {
+const post = ({uuid, html, published = false} = {}) => {
+	if(!uuid) {
+		throw Error('Missing required parameter [uuid]');
```

Comment:
Promise-returning functions shouldn't throw errors, as they won't always be caught by `.catch()`. We should reject instead.